src/main/kotlin/com/ajaxproject/warehouse/controller/ProductsController.kt

```diff
@@ -0,0 +1,18 @@
+package com.ajaxproject.warehouse.controller
+
+import com.ajaxproject.warehouse.service.ProductService
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/products")
+class ProductsController(val productService: ProductService) {
+
+    @GetMapping
+    fun findAllProducts() = productService.findAllProducts()
+
+    @GetMapping("/{id}")
+    fun findById(@PathVariable id: Int) = productService.findById(id)
```

Comment:
don't use implicit return type here.  
What does it returns? It's not clear.

**// OK**
```kotlin
fun function1() = 1 // it's clear

fun function2() = "some value" // it's clear

fun function3(): String = service.doSmthng()
```

**// NOT OK**
```kotlin
fun funtion3() = service.doSmthng() 

// I need to open the service to find out what does it return.
// Maybe there a code that returns something that is not clear as well.
```

Reply:
please, fix it over all the code